__mocks__/got.js

```diff
@@ -0,0 +1,73 @@
+const people = require('./../assets/people.json');
```

Comment:
My main concern with this is that you've mocked the `got` module. What happens when your code makes HTTP calls to other services in the future that also use the `got` module? How would you allow for that? What complications arise from this approach?

Reply:
If we need any functionality of the `got` module then we can use `jest.requireActual`

Ex
```
const newGot = jest.requireActual('got');
```

Reply:
So say in the future your app starts having to make other HTTP calls to non-SWAPI services. You're going to need to use `got` for those as well. Which means you're going to have to include those mocked calls in here as well. It's going to get messy fast.

What do you think is the best way to avoid that?

@VAIMORSS

Reply:
May be the Class structure!! @jdoyle65

Reply:
Sounds like you've got the idea now :)